flow/templates/stampede2.sh

```diff
@@ -35,7 +35,6 @@
 {% endif %}
 {% endblock %}
 
-{% block body %}
```

Comment:
I _think_ we still need the `body` block?

Reply:
Yes, you should still have that. Right now all of the logic for launcher is outside all blocks, which I think means that it won't render at all.